pkg/fanal/analyzer/language/nodejs/path.go

```diff
@@ -0,0 +1,27 @@
+package nodejs
```

Comment:
what if use `utils` file name for this file?

Reply:
done